pool/pool.go

```diff
@@ -17,6 +20,39 @@ type Pool struct {
 	unregisterChan chan types.Unregister
 	Map            map[uuid.UUID]types.API
 	name           string
+	ImageDB        *types.ImageDB
+}
+
+func (pool *Pool) GetImageDB() interface{} {
+	return pool.ImageDB
+}
+
+func setupImgDB() *types.ImageDB {
+	var imgDB types.ImageDB
+
+	imgFile, err := os.Open("./assets/images/images.json")
+
+	if err != nil {
+		log.Println(err.Error())
+		return &types.ImageDB{}
+	}
+
+	defer imgFile.Close()
+
+	byteValue, _ := ioutil.ReadAll(imgFile)
+
+	if err = json.Unmarshal(byteValue, &imgDB); err != nil {
+		log.Println(err.Error())
+		return &types.ImageDB{}
+	}
+
+	imgDB.NumImages = len(imgDB.Images)
+	return &imgDB
+}
+
+func SaveImageDB(db *types.ImageDB) {
```

Comment:
Why is this a package level function rather than a method of the `Pool` object? It seems odd to call it as `pool.SaveImageDB(p.ImageDB)` instread of calling `p.SaveImageDB() directly. Same with `setupImgDB()`.

Reply:
Mmm good thinking

Reply:
setupImgDB seems weird to have as a pool method, since its used in the pool's init. I think its probably ok having it as a seperate func for now?

Reply:
Maybe if we called it `CreateImageDB` or `NewImageDB` that would be better. Also if it's not connected to the pool it should probably be moved to the `types` package right under its interface declaration